supreme_court_predictions/statistics/descriptives.py

```diff
@@ -0,0 +1,246 @@
+"""
+This file houses the class that is used to provide descriptive summary 
+statistics for the convokit datasets.
+"""
+import os
+
+import numpy as np
+import pandas as pd
+
+from supreme_court_predictions.util.contants import ENCODING_UTF_8_SIG
+
+from .datacleaner import DataCleaner
+
+
+class Descriptives:
+    """
+    This class houses the functions needed to provide and export descriptive
+    statistics of the convokit data.
+    """
+
+    def __init__(self, downloaded_clean_corpus, save_data=True):
+        self.advocates_stats = None
+        self.conversations_stats = None
+        self.speakers_stats = None
+        self.utterances_stats = None
+        self.voters_stats = None
+
+        self.downloaded_clean_corpus = downloaded_clean_corpus
+        self.save_data = save_data
+
+        # Get local directory
+        cwd = os.getcwd()
+        self.local_path = (
+            cwd.replace("\\", "/")
+            + "/supreme_court_predictions/data/clean_convokit/"
+        )
+        print(f"Working in {self.local_path}")
+
+        # Set output path
+        self.output_path = self.local_path.replace(
+            "clean_convokit", "statistics"
+        )
+        print(f"Data will be saved to {self.output_path}")
+
+        # Download necessary data
+        if not downloaded_clean_corpus:
+            print("Cleaned corpus not present, downloading...")
+            cleaner = DataCleaner(True)
+            cleaner.parse_all_data()
+        else:
+            print("Cleaned corpus already downloaded")
+
+    def get_count_desc(self, df, cols):
+        """
+        [Insert]
```

Comment:
You got some of these bois to fill out.

Reply:
I know michael i am lazy, I'll do it tomorrow :p